### ISSUE TITLE
Instructions on installing.

### ISSUE THREAD

**Issue by Prakhar0409** [stremlau/html5_notifier#32]
Can someone please put some instructions in the README on how to install/setup this on chrome?

**Comment by surajshrikantjadhav** [comment]
just rename your config file and remove .dist extension from it in config directory of plugin
Mine worked this way

**Comment by rememberhim5** [comment]
Where do you put the files on your computer?  Do I have to use ftp?  Are there more instructions on how to install this notifier?

**Comment by meaz** [comment]
so you just git clone in plugins, and add yhis in config/config.inc.php:
`$config['plugins'] = array(
  'html5_notifier',
);`

right?

**Comment by spel-oe** [comment]
exactly, then it must show up when you click on the help-button near the "logout" 
now you can change the settings in the "Mailbox View" and test the plugin there
